Src/library/base/elks/kernel/tuple.e

```diff
@@ -836,6 +998,7 @@ feature -- Access
 				loop
 					if i > 1 then
 						l_type_string.append_character (',')
+						l_type_string.append_character (' ')
```

Comment:
Do you actually need the space in Gobo or is it just to make the type more readable in the debugger?